### ISSUE TITLE
Tilted atlas

### ISSUE THREAD

**Issue by enomis7401085** [mrymsadeghi/AMBIA#9]
We tried to tilt the atlas for matching a brain section. We activated tilted and changed numbers of affected quadrants and pressed "apply". Nothing happened, the section from the brain atlas remained at the same level? Do you have any suggestions how we can fix this problem?

Looking forward to hearing from you soon,

Best regards,
Simone

**Comment by amirbakhtiary23** [comment]
Hey there Simone,
That feature is not currently available, but we will release a new version in the very near future that leverages deep learning in this specific part of AMBIA. 
Stay tuned for new updates.
Regards,
Amir

**Comment by enomis7401085** [comment]
Dear Amri,

thank you for responding. I am a little bit confused because this is exactly what it was developed for and Francesco Ferraguti always promoted. I hope I remember correctly, but this feature used to work when Maryam validated  it together with Arnau from Francesco’s group. When will it be available again?

Best, simone

******************************************
Simone B. Sartori
Department of Pharmacology and Toxicology
CCB - Center for Chemistry and Biomedicine University of Innsbruck Innrain 80-82/3rd floor
A-6020 Innsbruck , AUSTRIA

tel: +43-512-507-58803
fax: +43-512-507-58899
e-mail: ***@***.***
homepage: http://www.uibk.ac.at/pharmazie/pharmakologie/sartori.html
FG-18: https://ANXIETYINASD-FG18.pages.ista.ac.at<https://anxietyinasd-fg18.pages.ista.ac.at/>
ESC 2025: https://www.psych.mpg.de/4th-esc
[ESC2025_Plakat_landscape+sponsors]

**Comment by enomis7401085** [comment]
When will it be available again? If I remember correctly, it was availble in an earlier vesion.

**Comment by amirbakhtiary23** [comment]
You are correct. It was available previously, though due to some issues we disabled this feature temporarily and we plan to release an update in the next month.

**Comment by amirbakhtiary23** [comment]
Hello @enomis7401085 .
The tilted atlas functionality has been added to the AMBIA.
You can check out the latest update.
Best,
Amir
